From 95c89dcd1b476b4925ecdaf6bad584d4be061fa2 Mon Sep 17 00:00:00 2001
From: LuK1337 <priv.luk@gmail.com>
Date: Wed, 1 Aug 2018 18:10:39 +0200
Subject: [PATCH 32/38] SystemUI: Initialize mClockVisibleByUser to
 android:visibility

* Fixes ~ triple clock ~ after leaving from Google SUW.

Change-Id: I3ceea4400b3057e2b6b1f221f9074b1d451133ea
---
 .../src/com/android/systemui/statusbar/policy/Clock.java        | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/SystemUI/src/com/android/systemui/statusbar/policy/Clock.java b/packages/SystemUI/src/com/android/systemui/statusbar/policy/Clock.java
index fe9bb942d2d..6b87bdec5c8 100644
--- a/packages/SystemUI/src/com/android/systemui/statusbar/policy/Clock.java
+++ b/packages/SystemUI/src/com/android/systemui/statusbar/policy/Clock.java
@@ -66,7 +66,7 @@ public class Clock extends TextView implements DemoMode, Tunable, CommandQueue.C
     public static final String CLOCK_STYLE = "lineagesystem:status_bar_am_pm";
 
     private boolean mClockVisibleByPolicy = true;
-    private boolean mClockVisibleByUser = true;
+    private boolean mClockVisibleByUser = getVisibility() == View.VISIBLE;
 
     private boolean mAttached;
     private Calendar mCalendar;
-- 
2.17.1

